### PR TITLE
Update go version to 1.16.3

### PR DESCRIPTION
**Pull request by cynepco3hahue**
Signed-off-by: Artyom Lukianov <alukiano@redhat.com>

### CODE REVIEW

**Comment by yuvalk** [comment]
what about:
```
$   head go.mod 
module github.com/openshift-kni/cnf-features-deploy

go 1.13
```
?

**Comment by cynepco3hahue** [comment]
> $   head go.mod 
> module github.com/openshift-kni/cnf-features-deploy
> 
> go 1.13

Fixed, thanks!

File: vendor/modules.txt [hunk 1]
```diff
@@ -24,6 +24,7 @@ github.com/coreos/go-json
 # github.com/coreos/go-semver v0.3.0
 github.com/coreos/go-semver/semver
 # github.com/coreos/go-systemd v0.0.0-20191104093116-d3cd4ed1dbcf
+## explicit
```
Comment (yuvalk) [inline vendor/modules.txt:27]: all this explicit comments came in automatically?
is that a new go feature?

Comment (cynepco3hahue) [reply vendor/modules.txt:27]: Yes, it came automatically. It a new go 1.14 feature, see - https://golang.org/doc/go1.14#go-command.

**Pull request merged**